docs/gangplank/usage.md

```diff
@@ -82,7 +82,20 @@ chattr +C file ~/workdir
 gangplank pod --workDir ~/workDir <options
 ```
 
-#### Secret Discovery
+### Remote Podman
+
+Gangplank supports the use remote Podman containers via the Podman GoLang bindings. Since the binding are controllable through envVars, Gangplank will blindly run remote containers in podman mode when `CONTAINER_HOST` is defined, athough Gangplank tries to be smart about it.
```

Comment:
```suggestion
Gangplank supports the use of remote Podman containers via the Podman GoLang bindings. Since the binding are controllable through envVars, Gangplank will blindly run remote containers in podman mode when `CONTAINER_HOST` is defined, although Gangplank tries to be smart about it.
```